impl/src/test/groovy/com/okta/sdk/impl/client/DefaultClientBuilderTest.groovy

```diff
@@ -93,6 +88,7 @@ class DefaultClientBuilderTest {
         DefaultClientBuilder clientBuilder = (DefaultClientBuilder) builder
         assertEquals clientBuilder.clientConfiguration.baseUrl, "https://api.okta.com/v42"
         assertEquals clientBuilder.clientConfiguration.connectionTimeout, 10
+        assertEquals clientBuilder.clientConfiguration.accessTokenTtl.seconds, 600
```

Comment:
Add tests for non duration syntax, i.e. if the user sets `accessTokenTtl = 3600`